keadm/cmd/keadm/app/cmd/common/types.go

```diff
@@ -87,8 +87,10 @@ type CollectOptions struct {
 }
 
 type ResetOptions struct {
-	Kubeconfig string
-	Force      bool
+	Kubeconfig  string
+	Force       bool
+	RuntimeType string
+	Endpoint    string
```

Comment:
Is it possible to wrap some common global flags as in kubeadm, so all the commands can use them instead of defining for each one :) @gy95